test/e2e/persistent_volume/persistent_volume_test.go

```diff
@@ -338,19 +342,29 @@ func createNfsProvisioner(svaName string, serverName string, scName string) {
 func deleteNfsRelatedRoles(serviceAccount string, clusterRole string, clusterRoleBinding string) {
 	ginkgo.By("Deleting NFS related roles and bindings")
 	err := kClient.DeleteClusterRoleBindings(clusterRoleBinding)
+	if err != nil && !k8serrors.IsNotFound(err) {
+		Ω(err).NotTo(HaveOccurred())
+	}
+
 	err2 := kClient.DeleteClusterRole(clusterRole)
-	err3 := kClient.DeleteServiceAccount(serviceAccount, dev)
+	if err2 != nil && !k8serrors.IsNotFound(err2) {
```

Comment:
Why assert only when k8serror is found?

Reply:
Thanks Ryan for helping me review!  `k8serrors.IsNotFound` returns true if the resource is not found
At line 147/148
147		createNfsRbac(saName, crName, crbName)
148		createNfsProvisioner(saName, serverName, scName)
At the beginning of the test, resources are created sequentially while some of them might be created successfully and some of not. We don't want to show resource not found error when cleaning up resources.

Reply:
Thanks for the prompt reply. It make total sense to me now!